src/core/modules/Planet.cpp

```diff
@@ -782,22 +782,26 @@ QString Planet::getInfoStringEloPhase(const StelCore *core, const InfoStringGrou
 		StelUtils::rectToSphe(&ra, &de, getEquinoxEquatorialPos(core));
 		StelUtils::equToEcl(raSun, deSun, obl, &lSun, &bSun);
 		StelUtils::equToEcl(ra, de, obl, &ecLong, &ecLat);
-		double elongAlongEcliptic=StelUtils::fmodpos(ecLong-lSun, M_PI*2.);
+		double elongationDecDeg, elongAlongEcliptic = StelUtils::fmodpos(ecLong-lSun, M_PI*2.);
 		if (elongAlongEcliptic > M_PI) elongAlongEcliptic-=2.*M_PI;
 
 		QString pha, elo, dLam;
+		bool sign;
+		StelUtils::radToDecDeg(elongAlongEcliptic, sign, elongationDecDeg);
```

Comment:
The function radToDecDeg is IMHO unneeded, and I remove its use it whereever I see it. Why dissect sign into a separate variable just to apply it immediately after once and never again? Just say 

elongationDecDeg=fmod(elongAlongEcliptic*M_180_PI, 360.); 

We have this (what feels like) thousands of times in our code. 

I will push a commit with some other changes I see useful.